azext_iot/tests/digitaltwins/test_dt_model_unit.py

```diff
@@ -186,7 +185,7 @@ def delete_request_callback(request):
                     models=None,
                     from_directory=ontology_directory,
                 )
-                assert len(models_added) == MAX_MODELS_PER_BATCH
+                assert len(models_added) == 30
```

Comment:
I think how it was before was better (referencing the variable rather than static integer)

Reply:
thanks will update!